### DISCUSSION TITLE
Is this an easter egg or something?

### DISCUSSION THREAD

**Discussion by zephiowl** [itzg/docker-minecraft-server#1760] [Q&A] [answered]
Hey so i'm running the docker mine-craft server on 1.19.2 spigot and whenever i look at the run command (htop) i can see the funky looking message saying "-DIReallyKnowWhatIAmDoingISwear" I'm assuming its an easter egg of the docker container but i'm not sure so i'm here to ask if it is. <3

![image](https://user-images.githubusercontent.com/66909997/193782596-5a816722-9f11-4483-a1b2-9bfa1184e31e.png)

**Comment by itzg** [comment, marked as answer]
It's to avoid a 20 second forced delay Spigot does when it thinks the server is out of date:

https://github.com/itzg/docker-minecraft-server/issues/417
